release-team/role-handbooks/communications/README.md

```diff
@@ -30,23 +30,35 @@ The expected time investment for both leads and shadows are as follows:
 
 **NOTE:** These are estimates and your personal experience may vary. The more time you can spend working as a team and regularly communicating within your team, the better off your experience will be.
 
+### External Release Communication
+
+Please use the `release-comms@kubernetes.io` Google Group list for external release communications (communicating with the CNCF, etc.).
+
+The following groups are members:
+* 1.18 Release Team Lead & Lead Shadows
+* 1.18 Communications Lead & Comms Shadows
+* SIG Release Chairs
+
+The list hast to be rotated/actively managed every cycle.
+
 ### Slack Channel
 
 There is a channel on the Kubernetes Slack workspace, `release-comms`, which is used by the communications release team to coordinate their efforts. If you're on the communications team, or applying to be, then it would be advantageous to follow along with the conversations.
 
 ## Release Milestone Activities
+Ensure to have the release team leads and shadows are notified while communicating with the CNCF.
```

Comment:
Remove this. It's explained above in the `External Release Communication` section.

Reply:
done